client/js/components/statement/splitStatement/CardPane.vue

```diff
@@ -139,7 +135,7 @@ export default {
       })
 
       Object.values(groupedCards).forEach(group => {
-        group.sort((a, b) => a.segment.charStart - b.segment.charStart)
+        // Group.sort((a, b) => a.segment.charStart - b.segment.charStart)
```

Comment:
Can this comment be removed?

Reply:
yes, done here https://github.com/demos-europe/demosplan-core/pull/4473/commits/ba741571752f908f15f4d656e8396d80c6a42ab2